src/utils.ts

```diff
@@ -202,3 +203,16 @@ export abstract class BaseCLICommand extends Command {
     return Promise.resolve()
   }
 }
+
+/**
+ * get token symbol by token address from config
+ * @param tokenContractAddress
+ * @returns {SupportedTokens} token symbol
+ */
+export function getTokenSymbol (tokenContractAddress: string): SupportedTokens {
```

Comment:
I am thinking of maybe to create `utils` in the `storage` service and move it there? As this is general app `utils` and this function is completely linked to the `storage` service? I will leave that up to your decision.

Reply:
Good idea. WIll move it there